src/defines.asm

```diff
@@ -255,8 +243,7 @@
 !ram_cm_custompalette_blue = !WRAM_MENU_START+$80
 !ram_cm_custompalette_green = !WRAM_MENU_START+$82
 !ram_cm_custompalette_red = !WRAM_MENU_START+$84
-!ram_cm_custompalette_hi = !WRAM_MENU_START+$86
-!ram_cm_custompalette_lo = !WRAM_MENU_START+$88
+!ram_cm_custompalette = !WRAM_MENU_START+$86
 !ram_cm_dummy_on = !WRAM_MENU_START+$8A
 !ram_cm_dummy_off = !WRAM_MENU_START+$8C
 !ram_cm_dummy_num = !WRAM_MENU_START+$8E
```

Comment:
The comment below this line could use an update.
> ; Reserve 48 bytes for CGRAM cache
; Currently first 22 bytes and last 2 bytes are used

Reply:
It wasn't as outdated as I thought.